Accompany/Controllers/Core/Home/HomeViewController.swift

```diff
@@ -9,35 +9,278 @@ import UIKit
+  private func setupLayout() {
+   
+    view.addSubview(accompanyTitleLabel)
+    view.addSubview(welcomeTitleLabel)
+    view.addSubview(stopWatchView)
+    
     
-    view.addSubview(button)
-    button.snp.makeConstraints { make in
-      make.centerX.equalTo(view.safeAreaLayoutGuide)
-      make.centerY.equalTo(view.safeAreaLayoutGuide)
+    accompanyTitleLabel.snp.makeConstraints { make in
+      make.top.equalTo(view).offset(60)
+      make.left.equalTo(view.safeAreaLayoutGuide).offset(40)
+      make.right.equalTo(view.safeAreaLayoutGuide).offset(-40)
+      
     }
+    
+    welcomeTitleLabel.snp.makeConstraints { make in
+      make.top.equalTo(accompanyTitleLabel.snp.bottom).offset(0)
+      make.left.equalTo(view.safeAreaLayoutGuide).offset(50)
+      make.right.equalTo(view.safeAreaLayoutGuide).offset(-50)
+    }
+    
+    stopWatchView.snp.makeConstraints { make in
+      make.right.equalTo(view.safeAreaLayoutGuide).offset(-30)
+      make.bottom.equalTo(view.safeAreaLayoutGuide).offset(-20)
+      make.width.equalTo(30)
+      make.height.equalTo(35)
+    }
+
+}
+  
+  private func setupNavButtonArray() {
+    
+    let stackView = UIStackView(arrangedSubviews: [button1, button2, button3, button4])
+    
+      stackView.axis = .vertical
+      stackView.alignment = .fill
+      stackView.distribution = .fill
+      stackView.spacing = 25
+      
+    view.addSubview(stackView)
+  
+    
+        stackView.snp.makeConstraints { make in
+          make.bottom.equalTo(view.safeAreaLayoutGuide).offset(-60)
+          make.left.equalTo(view.safeAreaLayoutGuide).offset(90)
+          make.right.equalTo(view.safeAreaLayoutGuide).offset(-90)
+        }
+    
+
   }
   
+
+  
   @objc func goToTodoList(_ button: UIButton) {
+    
     let todoListVC = TodoListViewController()
-    todoListVC.todoListTitleLabel.text = "1st Trimester"
+    if button == button1 {
+      todoListVC.todoListTitleLabel.text = "1st Trimester"
+      
+    } else if button == button2 {
+      todoListVC.todoListTitleLabel.text = "2nd Trimester"
+
+    } else if button == button3 {
+      todoListVC.todoListTitleLabel.text = "3rd Trimester"
+      
+    } else {
+      todoListVC.todoListTitleLabel.text = "After"
+    }
+    
+  
+  navigationController?.pushViewController(todoListVC, animated: true)
+  
+    
+  }
+  
+  @objc func gotoStopWatch(_ button: UIButton) {
     
-    navigationController?.pushViewController(todoListVC, animated: true)
   }
   
+ 
+
 }
 
+extension HomeViewController: UITableViewDelegate {
+  
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return todos.count
+  }
+  
+  
+}
+
+extension HomeViewController: UITableViewDataSource {
+  
+  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCell(withIdentifier: "NoteCellIdentifier", for: indexPath) as! NoteCell
```

Comment:
請把 "NoteCell" 改成 "TodoCell"。
```suggestion
    let cell = tableView.dequeueReusableCell(withIdentifier: TodoCell.identifier, for: indexPath) as! TodoCell
```